packages/client/tests/ecosystem/_utils/run.ts

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs/promises'
+import os from 'os'
+import path from 'path'
+import { $ } from 'zx'
+
+async function main() {
+  console.log('🎠 Preparing ecosystem tests')
+  // we first get all the paths we are going to need to run ecosystem
+  const tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), 'prisma-build'))
+  const cliPkgPath = path.join(__dirname, '..', '..', '..', '..', 'cli')
+  const clientPkgPath = path.join(__dirname, '..', '..', '..', '..', 'client')
+  const clientRuntimeDtsPath = path.join(clientPkgPath, 'runtime', 'index.d.ts')
+  const cliPkgJsonPath = path.join(cliPkgPath, 'package.json')
+  const clientPkgJsonPath = path.join(clientPkgPath, 'package.json')
+  const cliPkgJson = require(cliPkgJsonPath)
+  const clientPkgJson = require(clientPkgJsonPath)
+
+  // this process will need to modify some package.json, we save copies
+  await $`cd ${tmpDir} && cp ${cliPkgJsonPath} cli.package.json`.quiet()
+  await $`cd ${tmpDir} && cp ${clientPkgJsonPath} client.package.json`.quiet()
+  await $`cd ${tmpDir} && cp ${clientRuntimeDtsPath} client.runtime.d.ts`.quiet()
+
+  // we provide a function that can revert modified package.json back
+  const restoreOriginal = async () => {
+    await $`cd ${tmpDir} && cp cli.package.json ${cliPkgJsonPath}`.quiet()
+    await $`cd ${tmpDir} && cp client.package.json ${clientPkgJsonPath}`.quiet()
+    await $`cd ${tmpDir} && cp client.runtime.d.ts ${clientRuntimeDtsPath}`.quiet()
+  }
+
+  // if process is killed by hand, ensure that package.json is restored
+  process.on('SIGINT', () => restoreOriginal().then(() => process.exit(0)))
+
+  // use bundleDependencies to directly include deps like @prisma/engines
```

Comment:
Because we cannot publish to npm, we tell the package manager to `bundleDependencies`
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#bundledependencies